Modules/IO/MINC/test/itkMINCImageIOTest_2D.cxx

```diff
@@ -122,9 +123,8 @@ itkMINCImageIOTest_2D(int argc, char * argv[])
     }
     else
     {
-      std::cerr << "Incorrecte number of additional arguments " << std::endl;
-      std::cerr << "Usage: " << std::endl;
-      std::cerr << argv[0] << " inputfile outputfile [sum mx my ]" << std::endl;
+      std::cerr << "Incorrect number of additional arguments " << std::endl;
```

Comment:
Elsewhere you chnaged "argument" to "parameter".

Reply:
Changed in 79c65d7.